.github/workflows/styleguide.yml

```diff
@@ -0,0 +1,136 @@
+name: Generate Documentation Style Guide
+
+on:
+  # Run on installation (when workflow file is first created)
+  create:
+
+  workflow_call:
+    inputs:
+      commit-styleguide:
+        description: 'Commit the generated style guide to the repository'
+        required: false
+        default: true
+        type: boolean
+    secrets:
```

Comment:
no need to reference it here. instead use  ${{ secrets.OPENAI_API_KEY }}

Reply:
Is this required, as we have open_api_key in secrets in this same repo, wdyt? @HighnessAtharva